LandLord-latest/src/main/java/biz/princeps/landlord/manager/MobsManager.java

```diff
@@ -35,6 +37,12 @@ public MobsManager() {
         if (currentMineCraftVersion >= 1162) {
             register1_16_2Entities();
         }
+
+        if (currentMineCraftVersion >= 1171) {
+            register1_16_2Entities();
+        }
+
+        MOBS.sort(Comparator.comparing(iMob -> iMob.getType().name()));
```

Comment:
Why do we sort them now?

Reply:
Sorted entities are better for users imo, they will be sorted alphabetically in guis.